### PR TITLE
[FIX] Remove Close Button Panel for Poppy Intro

### PR DESCRIPTION
**Pull request by Droidze**
| Before | After |
|--------|--------|
| image | image |

### CODE REVIEW

**Review by eliasSFL — CHANGES_REQUESTED** [review]
Instead of putting the speaking modal in NPCModals you could add the close button panel into with the deals component of the flower bounties

File: src/features/world/ui/flowerShop/FlowerBounties.tsx [hunk 1]
```diff
@@ -55,26 +41,6 @@ export const FlowerBounties: React.FC<Props> = ({ readonly }) => {
...
-          acknowledgeIntro();
-          setShowIntro(false);
-        }}
-      />
-    );
-  }
-
   if (deal) {
     return (
       <Deal
```
Comment (eliasSFL) [inline src/features/world/ui/flowerShop/FlowerBounties.tsx:46]: So close button panel here

**Comment by Droidze** [comment]
> So close button panel here, and also at the return one below

@eliasSFL At the first thought about doing the easiest way, but also considered that you added `FlowerBounties` to Codex and could make issues there, so checked that 3 weeks ago you pulled out the `CloseButtonPanel` from `FlowerBounties` and added it to `NPCModals` for Poppy , probably it was not suitable to keep `CloseButtonPanel` inside. 

Briefly that's what happens when adding `CloseButtonPanel` inside `FlowerBounties`, shows it with two panels in Codex, which shouldn't be due to keeping consistency with other sections in this tab:

image

**Review by eliasSFL — APPROVED** [review]
I have refactored the flower bounties modal so that we can use one for poppy and one for the codex, and it will also solve the double modal issue

**Pull request merged**